recipes/marching_cubes/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "marching_cubes" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ilastik/marching_cubes/archive/{{ version }}.tar.gz
+  sha256: e173ca9a79382ff21f25dad5094133ad524d0958de773d8ea129a431d0a242de
+  patches:
+    # MSVC only supports openmp 2.0 syntax - there unsigned loop counter will
+    # not compile. Can be removed as soon as MSVC supports newer omp version
+    - 0001-ssize_t-in-parallel-fors-for-windows.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - boost-cpp
+    - pybind11
+    - libgomp  # [linux]
```

```suggestion
  build:
    - cmake
    - make  # [not win]
    - {{ compiler("cxx") }}
    - llvm-openmp  # [osx]
    - libgomp      # [linux]
  host:
    - python
    - boost-cpp
    - pybind11
```
https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp